tests/managers/user_tests.py

```diff
@@ -221,3 +220,20 @@ def test_create_user_handle_paas_exception(self):
             self.assertEqual(ex.args[0], "Your request for a new user was received, but it needs to be processed by "
                                          "the Platform Services API first. Barring any errors on the Platform Services "
                                          "side, your new user should be created shortly.")
+
+    # def test_list_user_filter(self):
+    #     test_filter = {'id': {'operation': 1234}}
+    #     self.manager.list_users(objectfilter=test_filter)
+    #     self.assert_called_with('SoftLayer_Account', 'getUsers', filter=test_filter)
+
+    def test_vpn_manual(self):
+        self.manager.vpn_manual(1234, True)
+        self.assert_called_with('SoftLayer_User_Customer', 'editObject', identifier=1234)
+
+    def test_vpn_subnet_add(self):
+        self.manager.vpn_subnet_add(1234, [1234])
+        self.assert_called_with('SoftLayer_Network_Service_Vpn_Overrides', 'createObjects')
+
+    def test_vpn_subnet_remove(self):
```

Comment:
for this and subnet_add tests, make sure the `assert_called_with` is checking the args as well.

Something like this (not sure about the exact ids)
`self.assert_called_with('SoftLayer_Network_Service_Vpn_Overrides', 'deleteObjects', args=({'id',11111, 'subnetId':1234}))`

Reply:
tests updated